cmd/test-server/kcp/shard.go

```diff
@@ -271,6 +271,32 @@ func (s *Shard) GatherMetrics(ctx context.Context) {
 	logger.Info("wrote metrics file", "path", metricsFile)
 }
 
+func ScrapeMetrics(ctx context.Context, s *Shard, workDir string) error {
+	logger := klog.FromContext(ctx).WithValues("shard", s.name)
+	promUrl, set := os.LookupEnv("PROMETHEUS_URL")
+	if !set {
+		logger.Info("PROMETHEUS_URL environment variable unset, skipping Prometheus scrape config generation")
+		return nil
+	}
+
+	logger.Info("scraping shard metrics using Prometheus")
```

Comment:
Likely want some informative keys here as to which shard, which prom

Reply:
done